game-app/game-headed/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -585,6 +586,7 @@ public void stopGame() {
     }
     bottomBar.setTerritory(null);
     GameShutdownRegistry.runShutdownActions();
+    StackTraceReportModel.setCurrentMapName(null);
```

Comment:
```suggestion
```
AFAIK setting the map name back to null should not be necessary. Is there a specific reason here?